sdk/communication/communication-administration/test/utils/recordedClient.ts

```diff
@@ -11,8 +11,10 @@ import {
   RecorderEnvironmentSetup,
   isPlaybackMode
 } from "@azure/test-utils-recorder";
-import { isNode } from "@azure/core-http";
+import { isNode, TokenCredential } from "@azure/core-http";
 import { CommunicationIdentityClient, PhoneNumberAdministrationClient } from "../../src";
+import { ClientSecretCredential } from "@azure/identity";
```

Comment:
Why do we use this, shouldn't we use https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/src/credentials/defaultAzureCredential.ts instead?

Reply:
done